contents.json

```diff
@@ -10989,10 +10990,7 @@
       "itunes": "https://itunes.apple.com/app/id1105827929",
       "lang": "rus",
       "screenshots": [
-        "http://a5.mzstatic.com/us/r30/Purple71/v4/d8/44/33/d844330e-cd1a-6612-f60a-36996fdcf4ce/screen696x696.jpeg",
-        "http://a2.mzstatic.com/us/r30/Purple71/v4/e6/80/41/e6804103-994c-26fd-9a57-af98e6dc4f16/screen696x696.jpeg",
-        "http://a1.mzstatic.com/us/r30/Purple71/v4/43/78/4c/43784ca8-f287-4666-38e6-5d33cc724695/screen696x696.jpeg",
-        "http://a2.mzstatic.com/us/r30/Purple71/v4/93/bb/5b/93bb5bdd-f09e-4d3f-c5d3-5eab08b7775e/screen696x696.jpeg"
+        "https://raw.githubusercontent.com/iAlexander/Homepok/master/Screenshots.jpg"
```

Comment:
Yes, I have changed it to GitHub hosted, because current ones will be deleted soon